getinvolved.qmd

```diff
@@ -5,15 +5,31 @@ toc: false
 
 ## DASL Guide
 
-The [WILDS Contributor Guide][guide] is the home for resources and best practices for WDL and Nextflow workflows, research compendia, Docker files and software.
+We maintain the [WILDS Contributor Guide][guide] as a collection of guidelines,
+review criteria, and best practices for developing production-ready WDL and 
+Nextflow workflows, research compendia, Docker files and software.
 
-In addition, the guide outlines the best practices that we strive to follow while developing WILDS.
+Our goal in developing the WILDS Contributor Guide is not just limited to the
+care and keeping of WILDS, we hope it can serve as a model for research or
+software development groups in terms of how they think about code and software
+quality.
 
 
 ## Make a contribution
 
-If you would like to make a contribution to any DaSL WILDS products please open a pull request in the respective GitHub repository. For example, if you want to make a contribution to the [sixtyfour][] package, open a pull request in the repository for that package at <https://github.com/getwilds/sixtyfour/>.
-
+If you would like to make a contribution to any DaSL WILDS projects please open 
+a pull request in the appropriate GitHub repository. For example, if you want to 
+make a contribution to the [sixtyfour][] package, open a pull request in 
+the repository for that package at <https://github.com/getwilds/sixtyfour/>.
+We are interested in accepting many different types of contributions. If you
+have not contributed to WILDS in the past, get in touch by emailing us and we
+would be happy to discuss the contribution you have in mind. Contributions in
+the form of reporting reproducable bugs, bug fixes, new  features, improved 
+documentation, additional software tests, translation and localization, blogs,
+or tutorials are all welcome.
+
+If you are interested in making another kind of contribution, we would be happy
+to chat. Please contact us at `wilds@fredhutch.org`.
```

Comment:
make this clickable so it opens up default email application? don't feel strongly just asking

Reply:
Good idea